examples/complete/main.tf

```diff
@@ -141,7 +143,7 @@ resource "time_sleep" "wait_30_seconds" {
 ##############################################################################
 
 module "read_only_replica_postgresql_db" {
-  count             = var.read_only_replicas_count
+  count             = 1 # There is a limit of five read-only replicas per leader
```

Comment:
Did you hardcode the count = 1 on purpose? Is it to keep a reference to an index? Normally a count = 1 is redundant, logically.

Reply:
Ive moved the read-only replica into the basic example (which runs in other_test.go), because I realised that it adds alot of extra time onto the complete example which runs in every PR. I moved this count back into a variable so it reads better